app/components/validate-request-purpose.js

```diff
@@ -0,0 +1,52 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
Can we make this component more re-usable by creating a separate component and using it as a mixin and providing these array of css classes. This can help us to this component in other forms validation in future. Thoughts ?

Reply:
sounds good @abulsayyad123 , i'll try it.